src/js/steps/initMonkey.js

```diff
@@ -21,6 +23,26 @@ module.exports = function ($events, options) {
 
     data.monkeyContainer.addClass('ready');
 
+    // Track errors on image load
+    var $images = data.monkeyContainer.find('[class*="page"] img');
+    $images.on('error', function () {
+      if (window.analytics) {
+        analytics.track('Broken image in monkey', { src: this.src });
+      }
+    });
+
+    var remaining = $images.filter(function () {
+      return !this.complete;
+    }).length;
+
+    $images.on('load', function () {
+      remaining--;
+
+      if (!remaining && window.analytics && options.platformAPI) {
+        analytics.track('Monkey images all loaded');
```

Comment:
Not sure how flexible this system is, but would it be possible to time how long it took to load them all? And maybe individual pages?